nautobot/ipam/tests/test_api.py

```diff
@@ -154,25 +163,45 @@ def setUpTestData(cls):
             virtual_machine=cls.test_vm,
             rd="65000:4",
         )
+        VRFDeviceAssignment.objects.create(
+            vrf=cls.vrfs[0],
+            virtual_device_context=cls.vdcs[0],
+            rd="65000:5",
+        )
+        VRFDeviceAssignment.objects.create(
+            vrf=cls.vrfs[0],
+            virtual_device_context=cls.vdcs[1],
+            rd="65000:6",
```

Comment:
Let's make sure we have diversity in the test data - one of these should probably set both `name` and `rd`, and the other should omit both?